.github/workflows/package.yml

```diff
@@ -17,13 +17,14 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
         with:
-          python-version: "3.13"
-      - name: Install Python dependencies
-        run: |
-          pip3 install build
+          python-version: '3.10'
```

Comment:
This looks like a separate fix for 38aae907fd065ee69c... can you submit separately?

Reply:
Yes, will remove from this PR.